superset-frontend/plugins/legacy-plugin-chart-country-map/src/countries.ts

```diff
@@ -154,6 +156,9 @@ export const countryOptions = Object.keys(countries).map(x => {
   if (x === 'italy_regions') {
     return [x, 'Italy (regions)'];
   }
+  if (x === 'turkey') {
```

Comment:
Thanks a lot for the contribution. might we keep the country name as English(Because many users cannot input non-ascii characters)?

Reply:
It was actually like that, but asked to changed to Türkiye on [`#20698`](https://github.com/apache/superset/pull/20698) and waits for review for 5 days :/

Reply:
Sorry for the waiting long time. I have approved. just waiting for the CI.

Reply:
@srinify could we keep the country name in English?

Reply:
I have changed it to Turkey